docs/json/schemas/event-schema.json

```diff
@@ -0,0 +1,75 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "title": "Event Schema",
+    "description": "The schema for the event from change feed to the orchestrator.",
+    "default": {},
+    "additionalProperties": true,
+    "required": ["id", "divisionId", "storeId", "upc", "inventoryCount", "productName", "lastUpdateTimestamp"],
+    "properties": {
+        "id": {
+            "$id": "#/properties/id",
+            "type": "string",
+            "title": "The Cosmos DB document id",
+            "description": "The format is divisionId:storeId:upc",
+            "examples": ["div1:store005:0003400029005"]
+        },
+        "divisionId": {
+            "$id": "#/properties/divisionId",
+            "type": "string",
+            "title": "The division id",
+            "examples": ["div1"]
+        },
+        "storeId": {
+            "$id": "#/properties/storeId",
+            "type": "string",
+            "title": "The store id",
+            "examples": ["store005"]
+        },
+        "upc": {
+            "$id": "#/properties/upc",
+            "type": "string",
+            "title": "The universal product code",
+            "examples": ["0003400029005"]
+        },
+        "countAdjustment": {
+            "$id": "#/properties/countAdjustment",
+            "type": "integer",
+            "title": "The current inventory of a given item",
+            "examples": [1000]
+        },
+        "type": {
+            "$id": "#/properties/type",
+            "type": "string",
+            "enum": ["shipmentUpdate", "onhandUpdate"],
```

Comment:
We just changed the event type to be camel case uniform with the rest of the project so it should now read
`"enum": ["shipmentUpdate", "onHandUpdate"],`

Reply:
Ok, I will update that.

Reply:
Done